### PR TITLE
Allow asPercent to take two series and match them together

### PR DESCRIPTION
**Pull request by cbowman0**
Convert asPercent to use matchSeries in order to provide the percentage of list1 to list2..

asPercent(carbon.carbon-daemons.*.cache0.writer.write_operations,carbon.carbon-daemons.*.cache0.writer.datapoints_written)

### CODE REVIEW

**Comment by cbowman0** [comment]
This was rebased against master.  Please let me know if there's anything else that's needed.

**Comment by obfuscurity** [comment]
:heart:

**Comment by obfuscurity** [comment]
It would be lovely to get tests for this change but I didn't want to hold this up any longer.

File: webapp/graphite/render/functions.py [hunk 1]
```diff
@@ -503,7 +507,7 @@ def changed(requestContext, seriesList):
         series[i] = 0
   return seriesList
 
-def asPercent(requestContext, seriesList, total=None):
+def asPercent(requestContext, seriesList, seriesList2orNumber=None):
   """
 
   Calculates a percentage of the total of a wildcard series. If `total` is specified,
```
Comment (obfuscurity) [inline webapp/graphite/render/functions.py]: Just realized the comments were not updated to reflect the removal of `total`. @cbowman0 would you mind fixing this?

Comment (obfuscurity) [inline webapp/graphite/render/functions.py]: Note that this was addressed in https://github.com/graphite-project/graphite-web/pull/1567.

**Pull request merged**